apps/ledger-live-desktop/src/renderer/screens/account/AccountBalanceSummaryHeader.jsx

```diff
@@ -43,7 +43,7 @@ export default function AccountBalanceSummaryHeader({
   const unit = getAccountUnit(account);
   const cvUnit = counterValue.units[0];
   const data = [
-    { valueChange: cryptoChange, balance: last.value, unit },
+    { valueChange: cryptoChange, balance: last.value, unit, dynamicSignificantDigits: 8 },
     { valueChange: countervalueChange, balance: last.countervalue, unit: cvUnit },
```

Comment:
You should add it here also no? 

cause below you use `data[1].dynamicSignificantDigits`

Reply:
For the other value, as it's a counter value, i would like to let it use the default value

Reply:
Ok but `data[1].dynamicSignificantDigits` doesn't exists so if you want the default value you need `dynamicSignificantDigits: 4` or i'm missing something?

Reply:
dynamicSignificantDigits is a parameter of formatCurrencyUnit (live-common), it can be undefined and it as a default value

Reply:
the default value is inside formatCurrencyUnit